Add `Featured Artist Track Updates: Krimek` news post (#13222)
ppy/osu-wiki: news/2025/2025-05-04-featured-artist-track-updates-krimek.md

---
layout: post
title: "Featured Artist Track Updates: Krimek"

> date: 2025-05-04 18:00:00 +0000

Reviewer: some spacing between posts

```suggestion
date: 2025-05-04 21:00:00 +0000
```

---

> From a regular in osu!'s [bespoke music](/wiki/Community/Bespoke_music) world to one of osu!'s Featured Artists, **Krimek** returns with a new batch of tracks!

Reviewer: the expression sounds a bit contrived

```suggestion
From a regular in osu!'s [music production scene](/wiki/Community/Bespoke_music) to one of osu!'s Featured Artists, **Krimek** returns with a new batch of tracks!
```

</audio>

### [Krimek feat. eili - Silent Sentinel](https://assets.ppy.sh/artists/371/Silent%20Sentinel/Krimek%20feat.%20eili%20-%20Silent%20Sentinel.osz)

Here's the [Vietnam osu! Championship 2024's Knockout 2 tiebreaker](https://osu.ppy.sh/beatmapsets/2254625) mapped by [Typ4](https://osu.ppy.sh/users/6902361), [Ducky-](https://osu.ppy.sh/users/9351565), [nik](https://osu.ppy.sh/users/10077264), [Nagabi](https://osu.ppy.sh/users/10352099), [Vermasium](https://osu.ppy.sh/users/11106442), and [Arushii](https://osu.ppy.sh/users/15664628)!

<audio controls>
    <source src="https://assets.ppy.sh/artists/371/Silent%20Sentinel/Krimek%20feat.%20eili%20-%20Silent%20Sentinel.mp3">
</audio>

### [Krimek - Revival Of A New Master](https://assets.ppy.sh/artists/371/Revival%20Of%20A%20New%20Master/Krimek%20-%20Revival%20Of%20A%20New%20Master.osz)

And here's [a map from Resurrection Cup 2023](https://osu.ppy.sh/beatmapsets/2028363) also mapped by the artist!

<audio controls>
    <source src="https://assets.ppy.sh/artists/371/Revival%20Of%20A%20New%20Master/Krimek%20-%20Revival%20Of%20A%20New%20Master.mp3">
</audio>

---

> If you're waiting for a sign before you start mapping, this is it. More specifically, every **NEW** tag beside songs on [**Krimek**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/371) is your sign. Pick one out and circle placing instead of circle clicking for a change.

Reviewer: ```suggestion
If you're waiting for a sign before you start mapping, this is it. More specifically, every **NEW** tag beside songs on [**Krimek**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/371) is your sign. Pick one out and start circle-placing instead of circle-clicking for a change.
```
